harismah/Dwella
# [TASK PR] Implement Edit Profile Page & Image Upload

**Description:**

This PR introduces the Edit Profile page, complete with frontend and backend integration to allow users to update their profile information. The current functionality includes fields for editing user details, and users can select an image from the public folder to update their profile picture. In future iterations, the image upload feature will be enhanced to utilize Supabase storage for secure and scalable image handling. This setup provides a foundation for the profile editing feature, with plans to expand image storage options in upcoming updates.

Testing Instructions:

1. Log in with valid credentials.
2. Navigate to the Profile page.
3. Click the Edit Profile button located to the right of the profile image.
4. Update profile information as desired and upload an image (select from the public folder).
5. Submit the changes and verify that the updated details are reflected on the Profile page.

These steps ensure the Edit Profile functionality works as expected and that changes are successfully saved and displayed.

Resolves Issue: #32 
Resolves Issue: #25
